index.js

```diff
@@ -0,0 +1,116 @@
+
+//player attributes
+const playerName = document.querySelector("h2")
+const playerImage = document.querySelector(".player img")
+const playerLikes = document.querySelector(".player p")
+const playerNickname = document.querySelector("em")
+//added player number to html
+const playerNumber = document.querySelector("#player-number")
+
+//page attributes
+const ul = document.querySelector("#goals")
+const liker = document.querySelector(".like-button")
+const videoForm = document.querySelector("#new-goal-form")
+
+
+//make network request to API
+fetch("http://localhost:3000/players/1")
+  //promise 1
+  .then(response => response.json())
+  //promise 2
+  .then(playerObj => {
+    //use data response from API
+    playerImage.src = playerObj.photo
+    playerImage.alt = playerObj.name 
+    playerName.textContent = playerObj.name 
+    playerNickname.textContent = playerObj.nickname
+    playerNumber.textContent = `Player Number: ${playerObj.number}`
+    playerImage.parentNode.dataset.id = playerObj.id 
+    playerLikes.textContent = `${playerObj.likes} Likes`
+    playerObj.goals.forEach(goal => {
+          renderGoal(goal)
+    })
```

Comment:
It may be helpful to create a separate function for this, just to clean up the code a bit.